rq/utils.py

```diff
@@ -297,3 +297,25 @@ def split_list(a_list, segment_size):
     """
     for i in range(0, len(a_list), segment_size):
         yield a_list[i:i + segment_size]
+
+
+def get_call_string(func_name, args, kwargs, max_length=None):
+    """Returns a string representation of the call, formatted as a regular
+    Python function invocation statement. If max_length is not None, truncate
+    arguments with representation longer than max_length.
+    """
+    if func_name is None:
+        return None
+
+    def truncate_long_string(data):
```

Comment:
One more thing, can you please move this to this module's top level so we can re-use this? The function signature should also change to `truncate_long_string(data, max_length=None):`

Reply:
Sure, no problem.

I took the liberty of creating an additional test since with that change `truncate_long_string` becomes a "public" function. I hope you're okay with that.

If there's anything else you think I could change in this pull request, I'm ready to help.